sentry_sdk/client.py

```diff
@@ -249,6 +253,11 @@ def _prepare_event(
             self.options["project_root"],
         )
 
+        if event is not None:
```

Comment:
Shall we run this after the `before_send` callbacks?

Reply:
we always said `before_send` sees what actually gets sent so let's not change that semantic